src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -589,6 +597,7 @@ PaError PaPulseAudio_Initialize( PaUtilHostApiRepresentation ** hostApi,
                 break;
             case PA_CONTEXT_TERMINATED:
             case PA_CONTEXT_FAILED:
+                result = paUnanticipatedHostError;
```

Comment:
I think there's a missing call to `PaUtil_SetLastHostErrorInfo` here. perhaps separate calls for each case.

Reply:
It's fine to do additional cleanup later, but this PR adds a `paUnanticipatedHostError`. Our policy is to not accept PRs that introduce new problems, so we'd appreciate it if you could fix this case in this PR please.

Reply:
Ok now it should be solved

Reply:
Thanks, but now the first parameter to `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` is not correct. It should be the native error code definitely not a PortAudio error code. For this PR, you could just set it to 0 if you don't have anything better. For further info check the definition of `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` in `pa_linux_pulseaudio_internal.h` and then the definition of `PaUtil_SetLastHostErrorInfo` in `pa_util.h`.

Also, `initiliaze` is mis-spelled on line 606.

While I'm here, note that you'll need to fix the `PA_PULSEAUDIO_SET_LAST_HOST_ERROR` in a later PR to pass the correct host api type.

Reply:
I've correct the errors and spelling and actually hostapi type is fixed in another PR but I can fix it here if it's needed.

Reply:
I've removed those new error codes and now it behaves as it was before only it goes to error (problem is that it just return `PaNoError` but as it did worked like that lately it should be ok.